Sources/SwiftDocC/Model/MarkdownOutput/Translation/MarkdownOutputMarkdownWalker.swift

```diff
@@ -0,0 +1,420 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Markdown
+
+/// Performs any markup processing necessary to build the final output markdown
+internal struct MarkdownOutputMarkupWalker: MarkupWalker {
```

Comment:
FYI (not blocking): it's redundant to specify `internal` here because it's the default access level.